test/util/compare-result.js

```diff
@@ -0,0 +1,18 @@
+const { expect } = require("chai");
+const jscodeshift = require("jscodeshift");
+const { compile } = require("coffee-script");
+
+module.exports = (testName, transform, source) => {
+  it(testName, () => {
+    const ES5Source = compile(source, { bare: true });
+    const transformedSource = transform({ ES5Source }, { jscodeshift });
```

Comment:
`{ ES5Source }` translates to `{ ES5Source: ES5Source }`. You probably meant `{ source: ES5Source }`. That along with `eval(ES5Source)` makes the test succeed.